### PR TITLE
tentative fix for mutiple curl commands

### PR DESCRIPTION
**Pull request by JonathanMontane**
Woops, sorry. I changed the indent to 4 to match the rest of our code base.

I'll make a comment on the real changes.

fixes issue #16

### CODE REVIEW

File: src/CurlParser.js [hunk 1]
```diff
@@ -3,526 +3,529 @@ import Immutable from 'immutable'
...
-      })
+
+    _parseCurlCommand() {
+        let request = new CurlRequest()
+        let urls = Immutable.List()
+        let arg
+        while ((arg = this._popArg()) != null) {
+            if (arg.toLowerCase() === 'curl') {
+                this._parseCurlCommand()
+            }
```
Comment (JonathanMontane) [inline src/CurlParser.js:144]: Before, we were not checking for the `curl` parameter in our conditions, therefore it fell in the final else (line:283), which generated a url from which it would create a request.

Therefore, the following commands:

``` sh
curl -X "GET" "https://echo.luckymarmot.com" \
    -H "abc: bab"
curl -X "GET" "https://echo.luckymarmot.com" \
    -H "abc: bcb"
```

would yield 3 requests with the following urls:
- https://echo.luckymarmot.com
- curl
- https://echo.luckymarmot.com

There's probably a better way to fix this, but it passes tests and since we're moving away from cURL-Importer in favor of API-Flow anyway, I don't really see the point of improving it more than that,

No tests were added, so it may fail miserably in some edge cases, but it should work most of the time

Comment (JonathanMontane) [inline src/CurlParser.js:144]: image

File: src/CurlParser.js [hunk 2]
```diff
@@ -3,526 +3,534 @@ import Immutable from 'immutable'
...
+        let arg
+        while ((arg = this._popArg()) != null) {
+            if (arg.toLowerCase() === 'curl') {
+                urls.forEach(url => {
+                    const r = request.set('url', url)
+                    this.requests = this.requests.push(r)
+                })
+                this.idx--
+                return
+            }
```
Comment (JonathanMontane) [inline src/CurlParser.js:144]: Before, we were not checking for the `curl` parameter in our conditions, therefore it fell in the final else (line:283), which generated a url from which it would create a request.

Therefore, the following commands:

``` sh
curl -X "GET" "https://echo.luckymarmot.com" \
    -H "abc: bab"
curl -X "GET" "https://echo.luckymarmot.com" \
    -H "abc: bcb"
```

would yield 3 requests with the following urls:
- https://echo.luckymarmot.com
- curl
- https://echo.luckymarmot.com

There's probably a better way to fix this, but it passes tests and since we're moving away from cURL-Importer in favor of API-Flow anyway, I don't really see the point of improving it more than that,

No tests were added, so it may fail miserably in some edge cases, but it should work most of the time

Comment (JonathanMontane) [inline src/CurlParser.js:144]: image

**Comment by hishnash** [comment]
can you add some test that try to parse this case?

**Comment by hishnash** [comment]
👍 Merge on green

**Comment by mittsh** [comment]
Cool!

@JonathanMontane can you make sure this bug doesn't happen in [API-Flow](https://github.com/luckymarmot/API-Flow)?
